CMakeLists.txt

```diff
@@ -315,6 +316,12 @@ if (BUILD_TESTING OR ENABLE_VIXL_DISASSEMBLER OR ENABLE_VIXL_SIMULATOR)
   include_directories(SYSTEM External/vixl/src/)
 endif()
 
+if (ENABLE_ZYDIS)
```

Comment:
Can you check for the library via `find_package` before falling back to the submodule, so that the system library is preferred? (Also see #2996 for motivation)

Reply:
OK, that's the solution atm then. find_package with fallback on submodule.